lang/resources/org/partiql/type-domains/partiql.ion

```diff
@@ -373,7 +373,7 @@ may then be further optimized by selecting better implementations of each operat
 
         (sum dml_op
             // `INSERT INTO <expr> <expr>`
-            (insert target::expr values::expr)
+            (insert target::expr values::expr conflict_action::(? conflict_action))
```

Comment:
The way I see it, it'll be like the following, but still need to think more about it:

```bash
(insert target::expr values::expr conflict_action::(? conflict_action) conflict_target::(? conflict_target))
```

Reply:
FWIW, the object model looks like something I can use.  Thanks.

Reply:
Because of conflict_action, do we need to update the statement?

https://github.com/partiql/partiql-lang-kotlin/blob/main/lang/resources/org/partiql/type-domains/partiql.ion#L690-L709

Reply:
Yes, that'll be in a consequent PR as pointed out in the PR overview. I decided to split the changes to reduce the review churn.